src/lib/core/CHIPEndPoint.h

```diff
@@ -0,0 +1,92 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file contains the EndPoint class
+ *
+ */
+
+#ifndef CHIPENDPOINT_H_
+#define CHIPENDPOINT_H_
+
+#include <lib/core/CHIPBaseAttribute.h>
+#include <lib/core/CHIPStandardClusters.h>
+
+namespace chip {
+namespace DataModel {
```

Comment:
once inside the chip namespace, it's unnecessary to use CHIP on the front of all the names.  can you trim these?

Reply:
Done